common/smoke-tests/pom.xml

```diff
@@ -0,0 +1,141 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>smoke-test</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>smoke-test</name>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+
+
+
+  <repositories>
+    <repository>
+      <id>azure-sdk-dev</id>
+      <url>https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-java/maven/v1</url>
+    </repository>
+  </repositories>
+
+
+  <profiles>
+    <!--
+     By default we build against our baseline, Java 8, but we also want to ensure compatibility
+     against the latest Java LTS release, currently Java 11. We therefore have two profiles:
+       * The default 'java8', which will perform a build using Java 8 as its target.
+       * The non-default 'java-lts' profile, which will build against the current LTS release.
+   -->
+    <profile>
+      <id>java8</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+        <property>
+          <name>!java-lts</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.8.1</version>
+            <configuration>
+              <source>1.8</source>
+              <target>1.8</target>
+              <showWarnings>true</showWarnings>
+              <showDeprecation>true</showDeprecation>
+              <compilerArgument>-Xlint:all</compilerArgument>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>java-lts</id>
+      <activation>
+        <property>
+          <name>java-lts</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.8.1</version>
+            <configuration>
+              <source>11</source>
+              <target>11</target>
+              <showWarnings>true</showWarnings>
+              <showDeprecation>true</showDeprecation>
+              <compilerArgument>-Xlint:all</compilerArgument>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+  </profiles>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12-beta-1</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.2.0-dev.20200107.1</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-cosmos</artifactId>
+      <version>4.0.0-preview.2</version>
```

Comment:
I'm not sure if we should be including Cosmos v4 in this just yet, at least not until they've moved into master and integrated with the centralized versioning.